packages/vue-components/src/Breadcrumb.vue

```diff
@@ -0,0 +1,95 @@
+<template>
+  <div>
+    <nav aria-label="breadcrumb">
+      <ol class="breadcrumb">
+        <li
+          v-for="(item, index) in items"
+          :key="item"
+          :class="['breadcrumb-item',
+                   {'active': isLast(index, items.length) || item.link === null},
+          ]"
+          :aria-current="{'page': isLast(index, items.length)}"
+        >
+          <a v-if="isLast(index, items.length) || item.link === null">
+            {{ item.title }}
+          </a>
+          <a v-else :href="item.link">
+            {{ item.title }}
+          </a>
+        </li>
+      </ol>
+    </nav>
+  </div>
+</template>
+
+<script>
+import normalizeUrl from './utils/urls';
+
+export default {
+  data() {
+    return {
+      items: [],
+    };
+  },
+  methods: {
+    isLast(index, length) {
+      return index === length - 1;
+    },
+  },
+  mounted() {
+    let siteNav = null;
+    let firstRootFound = false;
+
+    document.querySelectorAll('ul').forEach((el) => {
+      if (firstRootFound) {
+        return;
+      }
+      if (el.classList.contains('site-nav-list-root')) {
+        siteNav = el;
+        firstRootFound = true;
+      }
+    });
+
+    siteNav.querySelectorAll('a[href]').forEach((el) => {
```

Comment:
This is a pretty long block - for readability and future maintenance purposes, can we add some comments to explain what's going on here, or extract some operations into functions (whichever makes more sense)?

Reply:
> Nice work @itsyme 🥳
> 
> Is it possible to add some tests? e.g. generating breadcrumbs with different sitenavs or possible edge cases

Thanks for reviewing my PR @jovyntls! I'm currently working on the tests!